packages/search-ui/src/SearchDriver.js

```diff
@@ -3,6 +3,9 @@ import URLManager from "./URLManager";
 import RequestSequencer from "./RequestSequencer";
 import DebounceManager from "./DebounceManager";
 
+import { adaptFacets } from "./responseAdapter";
```

Comment:
I created these adapters to basically convert from UI State to App Search Query syntax. I'm actually going to move these to be applied in the App Search connector in a subsequent PR.

Right now, connectors convert from App Search query syntax to whatever their API syntax is. I want them to convert directly from UI State to whatever their API syntax is.